internal/accounts/account-add.go

```diff
@@ -0,0 +1,88 @@
+package accounts
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flow"
+	"github.com/onflow/flow-cli/pkg/flow/services"
+	"github.com/spf13/cobra"
+)
+
+type flagsAdd struct {
+	KeySigAlgo  string `default:"ECDSA_P256" flag:"sig-algo" info:"Signature algorithm"`
+	KeyHashAlgo string `default:"SHA3_256" flag:"hash-algo" info:"Hashing algorithm"`
+	KeyIndex    int    `flag:"index" info:"Account key index"`
+	KeyHex      string `flag:"privateKey" info:"Private key in hex format"`
+	KeyContext  string `flag:"context" info:"Projects/<PROJECTID>/locations/<LOCATION>/keyRings/<KEYRINGID>/cryptoKeys/<KEYID>/cryptoKeyVersions/<KEYVERSION>"`
+	Overwrite   bool   `flag:"overwrite,o" info:"Overwrite an existing account"`
+}
+
+var addFlags = &flagsAdd{}
+
+var AddCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "add <name> <address>",
+		Short:   "Add account by name to config",
+		Example: `flow accounts add alice 18d6e0586b0a20c5 --privateKey=11c5dfdeb0ff03a7a73ef39788563b62c89adea67bbb21ab95e5f710bd1d40b7`,
+		Args:    cobra.ExactArgs(2),
+	},
+	Flags: addFlags,
+	Run: func(
+		cmd *cobra.Command,
+		args []string,
+		project *flow.Project,
+		services *services.Services,
+	) (command.Result, error) {
+		account, err := services.Accounts.Add(
+			args[0],
+			args[1],
```

Comment:
It could be nice to assign these args to variables, or add a comment, just so we know what they are (same goes for all `args[X]` across all commands)

Reply:
I'm afraid if I assign them to variable it might happen someone else might see that as a good place to do additional logic on those args, so maybe just comments for now